### ISSUE TITLE
List of topics

### ISSUE THREAD

**Issue by taku-y** [PyDataOsaka/handson_pytorch#2]
The below list will be reorganized.

* Basic topics
  * [ ] Show pytorch version
  * [ ] Fitting linear model
    * [ ] Create tensors (and variables?)
    * [ ] Tensor operations
    * [ ] Tensor shape
    * [ ] Define a model ([link](https://pytorch.org/docs/stable/nn.html#module))
      * ([Define the network](https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html#define-the-network))
    * [ ] Loss function ([link](https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html#loss-function))
    * [ ] Optimize model parameters
      * ([Backprop](https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html#backprop))
      * ([Update the weights](https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html#update-the-weights))
  * [ ] Layers (models) and optimizers
  * [ ] Save/load parameters ([link](https://pytorch.org/tutorials/beginner/saving_loading_models.html#saving-loading-model-for-inference))
  * [ ] Visualization with tensorboard ([link](https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html#visualizing-models-data-and-training-with-tensorboard))
* Advanced topics
  * [ ] Checkpoint optimizer's internal state ([link](https://pytorch.org/tutorials/beginner/saving_loading_models.html#saving-loading-a-general-checkpoint-for-inference-and-or-resuming-training))
  * [ ] VAE
  * [ ] Quantile regression ([link](https://gist.github.com/taku-y/471b7eae5ef85badb1ddcfb389982bd6))
* References
  * [PyTorch tutorials](https://pytorch.org/tutorials/)
